Makefile

```diff
@@ -82,7 +82,7 @@ test: test-backend test-frontend
 .PHONY: build-backend
 build-backend: fmt-backend
 	@echo "### Building backend"
-	go build -ldflags "-X 'main.buildVersion=${BUILD_VERSION}' -X 'main.buildDate=${BUILD_DATE}'" -mod vendor -o plugin-backend cmd/plugin-backend.go
+	go build -gcflags='-N -l' -ldflags "-X 'main.buildVersion=${BUILD_VERSION}' -X 'main.buildDate=${BUILD_DATE}'" -mod vendor -o plugin-backend cmd/plugin-backend.go
```

Comment:
Do you guys also need this to attach debug in `vscode` ?
Else some symbols are not resolved :thinking:

Reply:
let's go for a variable. Thanks